resourcegroup/Program.cs

```diff
@@ -115,8 +115,9 @@ public static async Task Main(string[] args)
             var subscriptionId = secretServicePrincipalSettings.GetValue("subscriptionId").ToString();
             var resourceManagerEndpointUrl = secretServicePrincipalSettings.GetValue("resourceManagerEndpointUrl").ToString();
             var location = secretServicePrincipalSettings.GetValue("location").ToString();
+            var disableInstanceDiscovery =  ((bool)secretServicePrincipalSettings.GetValue("disableInstanceDiscovery"));
```

Comment:
Does this line throw an error if parameter is missing? Currently some of the other JSON configs does not require this for ASH and does it not have it.